routing/pathfind.go

```diff
@@ -606,7 +607,24 @@ func findPath(tx *bolt.Tx, graph *channeldb.ChannelGraph,
 			edgeInfo *channeldb.ChannelEdgeInfo,
 			outEdge, _ *channeldb.ChannelEdgePolicy) error {
 
-			processEdge(outEdge, edgeInfo.Capacity, pivot)
+			// We'll query the lower layer to see if we can obtain
+			// any more up to date information concerning the
+			// bandwidth of this edge.
+			var (
+				edgeBandwidth lnwire.MilliSatoshi
+				ok            bool
+			)
+			edgeBandwidth, ok = bandwidthHints[edgeInfo.ChannelID]
```

Comment:
Style nit: Looks like we can just declare these here rather than within the `var ()` above.